src/main/resources/application.yaml

```diff
@@ -0,0 +1,42 @@
+server:
+  port: 8081
+spring:
+  application:
+    name: pw-parking-permit
+#----------------------------------------
+# Open API
+#----------------------------------------
+openapi:
+  name: ${spring.application.name}
+  title: ${spring.application.name}
+  version: '@project.version@'
+
+#----------------------------------------
+# Camunda
+#----------------------------------------
+camunda:
+  bpm:
+    client:
+      base-url: http://localhost:8080/engine-rest
```

Comment:
This should not be specified here, it belongs in the .env-file (or in the specific environment yaml file)

Reply:
From camunda-template. We should change there aswell.